spacewalk/setup/spacewalk-setup.spec

```diff
@@ -295,18 +276,19 @@ make test
 %doc Changes README answers.txt
 %config %{_sysconfdir}/salt/master.d/susemanager.conf
 %config %{_sysconfdir}/salt/master.d/salt-ssh-logging.conf
+%config %{_sysconfdir}/tomcat/conf.d/tomcat_java_opts.conf
 %{perl_vendorlib}/*
 %{_bindir}/spacewalk-setup
 %{_bindir}/spacewalk-setup-httpd
 %{_bindir}/spacewalk-make-mount-points
 %{_bindir}/spacewalk-setup-cobbler
-%{_bindir}/spacewalk-setup-tomcat
 %{_bindir}/spacewalk-setup-sudoers
-%{_bindir}/spacewalk-setup-ipa-authentication
 %{_mandir}/man[13]/*.[13]*
 %dir %attr(0755, root, root) %{_prefix}/share/salt-formulas/
 %dir %attr(0755, root, root) %{_prefix}/share/salt-formulas/states/
 %dir %attr(0755, root, root) %{_prefix}/share/salt-formulas/metadata/
+%dir %attr(0755, root, root) %{_sysconfdir}/tomcat/
+%dir %attr(0755, root, root) %{_sysconfdir}/tomcat/conf.d/
```

Comment:
This conflicts with the Tomcat package (at least for EL packages):

```
Error: Transaction test error:
  file /etc/tomcat conflicts between attempted installs of spacewalk-setup-4.4.7-220400.9.1.uyuni2.noarch and tomcat-1:9.0.65-2.el9.noarch
  file /etc/tomcat/conf.d conflicts between attempted installs of spacewalk-setup-4.4.7-220400.9.1.uyuni2.noarch and tomcat-1:9.0.65-2.el9.noarch
```
As far as I understand the `Java:packages/tomcat` package on OBS, this should also be the case for SUSE installations.

Technically the folder belongs to Tomcat and should be owned by that package. So theoretically, this package should `Requires: tomcat` (but not `BuildRequires`). Practically, tomcat is installed via dependencies of dependencies.

Created a PR: https://github.com/uyuni-project/uyuni/pull/6841